src/java.base/windows/classes/sun/nio/fs/WindowsPath.java

```diff
@@ -206,7 +206,7 @@ String getPathForWin32Calls() throws WindowsException {
         // directory on removal media devices can change during the lifetime
         // of the VM)
         if (type != WindowsPathType.DRIVE_RELATIVE) {
-            synchronized (path) {
+            synchronized (this) {
```

Comment:
To be honest, I cannot see why this `synchronized` block exists at all...

Reply:
It's not strictly needed and could be removed.